src/components/Version.tsx

```diff
@@ -1,37 +1,41 @@
 import React, { FC } from "react";
 import { Icon, Tooltip } from "@canonical/react-components";
 import { useSettings } from "context/useSettings";
+import { RECENT_MAJOR_SERVER_VERSION, UI_VERSION } from "util/version";
 
-const ServerVersion: FC = () => {
+const Version: FC = () => {
   const { data: settings } = useSettings();
 
-  const version = settings?.environment?.server_version;
-  if (!version) {
+  const serverVersion = settings?.environment?.server_version;
+  if (!serverVersion) {
     return null;
   }
 
-  const major = version.includes(".") ? version.split(".")[0] : undefined;
-  const recentMajor = 5;
-  const isOutdated = major ? parseInt(major) < recentMajor : false;
+  const serverMajor = serverVersion.includes(".")
+    ? serverVersion.split(".")[0]
+    : undefined;
+  const isOutdated = serverMajor
+    ? parseInt(serverMajor) < RECENT_MAJOR_SERVER_VERSION
+    : false;
 
   return (
     <>
       <hr className="p-side-navigation__list is-dark navigation-hr" />
       <li className="p-side-navigation__link server-version">
         {isOutdated && (
           <Tooltip
-            message="You are using an outdated version.
+            message="You are using an outdated server version.
 Update your LXD server to benefit from the latest features."
             tooltipClassName="version-warning"
             zIndex={1000}
           >
             <Icon name="warning" className="p-side-navigation__icon" />
           </Tooltip>
         )}
-        Server {version}
+        Version {serverVersion}-ui-{UI_VERSION}
```

Comment:
I thought we had agreed on the shorthand "Ver"?
```suggestion
        Ver {serverVersion}-ui-{UI_VERSION}
```

Reply:
I looked at the "Ver" before, and it seemed a bit unclear what the string even presents. The version gibberish after that is already quite cryptic. We have enough space to write out "Version", so maybe keep it as is?

Reply:
Might be. It's just that it looks a bit tight in space on my small-width screen:

![image](https://github.com/canonical/lxd-ui/assets/56583786/b72b5913-2f3e-4041-b7ac-c474923dd6da)

Maybe ask @piperdeck if this is good? It might just be me, but I'd like to have a bit more margin on the right. It's also relevant to know if this can become any longer, e.g. 5.17-ui-5-1 might fit, but what if we have 8.88-ui-8-88? (picking 8 here since it's the widest number)

Reply:
as discussed, using a much smaller font to give it more margin.